renderers/team_renderer.py

```diff
@@ -104,13 +104,24 @@ def render_team_details(cls, handler, team, year, is_canonical):
                         team_rank = ranking['rank']
                         break
 
+            video_ids = ""
+            for lv in Match.COMP_LEVELS:
+                ms = matches_organized[lv]
+                for m in ms:
+                    vidstr = ""
+                    for v in m.youtube_videos:
+                        if len(v) > 0:
+                            vidstr = vidstr + v.split("?")[0] + ","
+                    video_ids = video_ids + vidstr
```

Comment:
Untested, but you could probably simplify this to something like (also made variables more verbose for readability's sake):

```python
video_ids = []
for level in Match.COMP_LEVELS:
    matches = matches_organized[level]
    for match in matches:
        video_ids += [video.split("?")[0] for video in match.youtube_videos]

# ...

"video_ids": ",".join(video_ids)

# or maybe even better,

"playlist": "https://www.youtube.com/watch_videos?video_ids={}&title={}".format(",".join(video_ids), event.name)
```

Reply:
I like the second one, ill implement/test that soon